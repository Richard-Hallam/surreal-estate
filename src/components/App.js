import '../styles/App.css';
import React from 'react';

const App = (props) => {
  return (
    <h2>Surreal estate</h2>

  );
};

export default App;
