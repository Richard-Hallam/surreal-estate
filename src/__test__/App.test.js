import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import App from '../components/App';


describe("app", () => {
    const { asFragment } = render(<App />);

    it("renders correctly", () => {
        expect(asFragment()).toMatchSnapshot();
    });

    it("renders heading text correctly", () =>{
      render(<App />);
      const linkElement = screen.getByText(/Surreal estate/i);
      expect(linkElement).toBeInTheDocument();
    })
});
