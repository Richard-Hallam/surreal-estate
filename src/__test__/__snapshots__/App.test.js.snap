// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app renders correctly 1`] = `
<DocumentFragment>
  <h2>
    Surreal estate
  </h2>
</DocumentFragment>
`;
